policies/sdc_approve.rego

```diff
@@ -0,0 +1,31 @@
+package karavi.sdc.approve
+
+import data.karavi.common
+
+# Allow requests by default.
+default allow = true
+
+default response = {
+	"allowed": true
+}
+response = {
+    "allowed": false,
+    "status": {
+        "reason": reason,
+    },
+} {
+    reason = concat(", ", deny)
+    reason != ""
+}
+
+deny[msg] {
+  common.roles == {}
+  msg := sprintf("no role data found", [])
```

Comment:
deny rules needs change, role data is not necessary

Reply:
removed role data